verify/db.go

```diff
@@ -43,22 +73,31 @@ func (db *DB) AddKey(id string, k *data.Key) error {
 	return nil
 }
 
-var validRoles = map[string]struct{}{
+var topLevelRoles = map[string]struct{}{
```

Comment:
Hmm. I'm not crazy about go-tuf inheriting the bad design decision from the original python-tuf of keeping keys in a module-wide dictionary, but this is not our doing.